conf/routes

```diff
@@ -56,7 +56,7 @@ GET         /photo/encounter/:id                               @femr.ui.controll
 GET         /triage/:id                                        @femr.ui.controllers.TriageController.indexPopulatedGet(id: Integer)
 GET         /triage                                            @femr.ui.controllers.TriageController.indexGet()
 POST        /triage                                            @femr.ui.controllers.TriageController.indexPost(id: Integer)
-POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer)
+POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer, deleteByUserID: Integer)
```

Comment:
Hi Kevin, I’ve modified my code per your request below and it is working now.

Thanks,
Vivek

From: Kevin Zurek [mailto:notifications@github.com]
Sent: Sunday, February 21, 2016 2:09 PM
To: FEMR/femr
Cc: ao3742
Subject: Re: [femr] track user deletes patient (#484)


In conf/routes<https://github.com/FEMR/femr/pull/484#discussion_r53571297>:

> @@ -56,7 +56,7 @@ GET         /photo/encounter/:id                               @femr.ui.controll

>  GET         /triage/:id                                        @femr.ui.controllers.TriageController.indexPopulatedGet(id: Integer)

>  GET         /triage                                            @femr.ui.controllers.TriageController.indexGet()

>  POST        /triage                                            @femr.ui.controllers.TriageController.indexPost(id: Integer)

> -POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer)

> +POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer, deleteByUserID: Integer)

Adding a route is one way to do this, but the deletePatientPost() method has a currentUser variable which will always contain the ID of the user (just use currentUser.getId())

—
Reply to this email directly or view it on GitHub<https://github.com/FEMR/femr/pull/484/files#r53571297>.